frontend/src/toolbar/actions/ActionsTab.scss

```diff
@@ -1,9 +1,17 @@
 .action-block-body {
-    padding: 16px;
+    form {
+        .form-error {
+            color: var(--danger);
+        }
+
+        //// TODO why doesn't this match?
```

Comment:
Can we remove this entirely?

Reply:
me: "I'll definitely remember to check for TODOs before I'm finished"
:side-eye-monkey-dot-meme: